### ISSUE TITLE
TCGA-lung: please share the training-validation / test split

### ISSUE THREAD

**Issue by GeorgeBatch** [binli123/dsmil-wsi#67]
Hi Bin,

I got the summary of what you kindly made available:

* [TCGA multiscale patches](https://drive.google.com/file/d/17zCn-WRNzxxxh8kkdBTbDLDZy0XZ3RIu/view?usp=sharing) (2.5x and 10x magnification)

* [Embedders pre-trained with SimCLR on TCGA patches](https://drive.google.com/drive/folders/1Rn_VpgM82VEfnjiVjDbObbBFHvs0V1OE)
    * magnification: low=2.5x, high=10x
    * pre-taining: v0 for 3 days, v1 for 2 weeks

* [TCGA multiscale features](https://drive.google.com/drive/folders/1v0ZEgSIYgriYuRn_O2p0t3RXdFHtiA61) (patches were passed through the v1 pre-trained embedders)

**Can you please share which subset of the patches (which WSIs) you used to train the Embedder?** I would like to try a different aggregation model on the same [TCGA multiscale features](https://drive.google.com/drive/folders/1v0ZEgSIYgriYuRn_O2p0t3RXdFHtiA61). But it would only make sense to train-validate the model on the same train-validation set as you did and test the model on a separate test set since, as I understand, during the SimCLR pre-training, you have exposed the embedder only to the train-validation portion of the TCGA and kept the test set truly unseen even to an embedder.

Many thanks,
George

**Comment by PiumiDS** [comment]
Hi @GeorgeBatch ,

Can you please let me know if you have tried to download the [TCGA multi-scale patches](https://drive.google.com/file/d/17zCn-WRNzxxxh8kkdBTbDLDZy0XZ3RIu/view?usp=sharing) zip file successfully as I'm repeatedly running into errors while downloading.
Many thanks for your kind reply.

**Comment by da-nial** [comment]
Hi @GeorgeBatch 
Hope you're doing well. Did you by any chance find out the train/validation/test split that was used to train the TCGA embedder? I contacted Bin, but unfortunately, they don't remember this as it was far in the past. 

Thanks in advance.

**Comment by GeorgeBatch** [comment]
Hi @da-nial, unfortunately, I got the same reply. I am working under the assumption that the test set specified [here](https://drive.google.com/drive/folders/1UobMSqJEqINX2izxrwbgprugjlTporSQ) was not used in the self-supervised training. So, I used the same split in my experiments when using the self-supervised feature extractor.
